mysql-test/suite/sys_vars/t/innodb_encrypt_tables_basic.test

```diff
@@ -1,24 +0,0 @@
-SET @@GLOBAL.innodb_encrypt_tables="OFF";
```

Comment:
How happen `mysql-test/suite/sys_vars/t/innodb_encrypt_tables_basic.test` is deleted whereas
`mysql-test/suite/sys_vars/r/innodb_encrypt_tables_basic.result` is updated. in the same commit?

Reply:
innodb_encrypt_tables.test and .result should be removed, it no longer exists

Reply:
I cherry picked the 5.7 commit, which updated these files. Then I noticed that the variable is gone, so I deleted the files - maybe I used `rm` instead of `git rm` for the result file. Fixed.